lib/rofl_ofdpa_fm_driver.cpp

```diff
@@ -840,15 +817,17 @@ void rofl_ofdpa_fm_driver::disable_policy_specific_lacp(
   fm.set_match().set_eth_dst(rofl::cmacaddr("01:80:c2:00:00:02"));
   fm.set_match().set_in_port(in_port);
 
-  DEBUG_LOG(": send flow-mod:" << std::endl << fm);
+  DEBUG_LOG(": return flow-mod:" << std::endl << fm);
 
-  dpt.send_flow_mod_message(rofl::cauxid(0), fm);
+  return fm;
 }
 
-void rofl_ofdpa_fm_driver::enable_policy_dhcp(rofl::crofdpt &dpt) {
+rofl::openflow::cofflowmod
+rofl_ofdpa_fm_driver::enable_policy(uint8_t ofp_version, int16_t src_port,
```

Comment:
I have to nitpick again, something like enable_policy_broadcast_udp would be more suitable here since just enable_policy is too generic.